drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -194,10 +208,14 @@ def getPropertyMap(line, property_names):
 	property_values = line.split("\t")
 
 	for i in range(0, len(property_names)): #do not exclude first col (filename), the schema checks for it
-		##remove trailing newline, and replace space with underscore
+		## remove trailing newline, and replace space with underscore
 		name = property_names[i].rstrip('\n').replace(" ", "_")
 		value = property_values[i].rstrip('\n').replace(" ", "_")
 
+		# look for placeholder symbol to skip property ("*")
+		if value == "*":
+			continue
```

Comment:
NIT: I'd suggest making this into a dedicated variable with a small line of explanation where the placeholder is applicable
```suggestion
		# look for placeholder symbol to skip property ("*")
		if value == "*":
			continue
```

Reply:
Good point, I added a global `PROPPERTY_PLACEHOLDER` constant, with a line explaining the meaning

Reply:
Nice 🆗